cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -13,49 +13,51 @@ Since full-text indexes are not automatically used by Cypher, this does not lead
 Users need to explicitly call procedures to use these indexes.
 The problem is only that if this behavior is not understood by the user, they might expect the full text index to return the same results that a different, but semantically similar, Cypher query does.
 
-==== Example with denylisted properties
+==== Example with blacklisted properties
```

Comment:
As David mentioned, we are making a conscious effort of not using the word `blacklisted` per https://trello.com/c/DhCCoW4i/2713-replace-whitelist-blacklist-with-allowlist-denylist. Is there a particular reason we should revert to this?

Reply:
No, I don't particularly care either way. I just remembered when we updated the hospital example for [fine-grained graph privileges in the operations manual](https://neo4j.com/docs/operations-manual/current/authentication-authorization/access-control/#auth-access-control) and I think `denylist` had been changed to `blacklist` in there, hence updating it here as well. I can drop the second commit if we prefer to keep it how it was

Reply:
I suggest we use `blocked` in this page, in place of either `blacklisted` or `denylisted`

@Hunterness do you know if there are any problems with using _blocked_/_blocking_ verbiage in this example (as opposed to _blacklisted_/_blacklisting_)?

Another alternative is _denied_/_denying_ - I think we should try to avoid using _denylisted_/_denylisting_ as outside of usage in the command it looks and sounds odd!

Reply:
I agree that _denylisted/denylisting_ sounds odd, one of the reasons I changed it.

I think _denied/denying_ is better then _blocked/blocking_, will update together with the other review comments :)